storage.js

```diff
@@ -0,0 +1,89 @@
+const funcs = [];
+funcs.push(require('./models/users.js'));
+funcs.push(require('./models/challenges.js'));
+funcs.push(require('./models/roles.js'));
+funcs.push(require('./models/user_roles.js'));
+funcs.push(require('./models/role_permissions.js'));
+
+const Schema = require('caminte').Schema
+
+memoryUpdate = function (model, filter, data, callback) {
```

Comment:
Scoping without const here intended?
```suggestion
memoryUpdate = (model, filter, data, callback) => {
```

Reply:
should have a const